bot/resources/tags/mcintent.md

```diff
@@ -0,0 +1,16 @@
+---
+embed:
+    title: "Message Content Intent"
+---
+
+Discord regulates data being sent to an application via a specific number.
+
+This number represents different permissions A.K.A. intents.
```

Comment:
I think the explanation of how the gateway intents bitfield is implemented is necessarily important here - we want to explain *how to do something* rather than *how something works*, if that makes sense. So something like
```suggestion
The discord gateway only dispatches events you subscribe to, which you can configure by using "intents"
```